### ISSUE TITLE
saving values in the optional assessments

### ISSUE THREAD

**Issue by sebakoe** [icra/ecam#463]
Is it possible to save the entered values / choosen options in the optional assessments when switching from YES to NO to YES? (e.g. via the cache?!)

e.g. assessing truck transport (in this case it only one value, but other optional assessments are more complex)

![image](https://user-images.githubusercontent.com/84188310/124136798-61b9bc80-da85-11eb-953d-a3989f03c863.png)
![image](https://user-images.githubusercontent.com/84188310/124136861-71d19c00-da85-11eb-9e70-d52e49b330fb.png)
![image](https://user-images.githubusercontent.com/84188310/124136905-7bf39a80-da85-11eb-87cb-7e4e17729965.png)

**Comment by holalluis** [comment]
yes, this is the expected behaviour, sorry I was doing some testing (for language translation) and I enabled all the questions to "yes" by default (just in development mode), when you save a file all the "yes" or "no" options will be saved inside the json file

**Comment by holalluis** [comment]
please check now, the "no" and "yes" selections should be remembered when loading a file

**Comment by sebakoe** [comment]
that was not the issue, sorry my explanation was not good enough. The general saving works fine. 

What I meant was that when after entering a value and turning the optional assessment off and on again, the previously entered values will be erased. So testing different scenerios is time consuming, you either have to save often or to re-enter the values. So, it would be good the the entered values are kept even if the option is turned off again.

**Comment by holalluis** [comment]
ah ok, now I understand. I have to come up with a way to solve this. I think that a button to duplicate a substage would work, because then would be easy to compare, right?

**Comment by sebakoe** [comment]
that work around might be ok - and the possibility of duplicating a substage, can save some time in general. I just realized that one can duplicate the entire assessment.
